sklearn/svm/tests/test_svm.py

```diff
@@ -691,19 +728,6 @@ def test_immutable_coef_property():
                       clf.coef_.__setitem__, (0, 0), 0)
 
 
-def test_inheritance():
-    # check that SVC classes can do inheritance
-    class ChildSVC(svm.SVC):
-        def __init__(self, foo=0):
-            self.foo = foo
-            svm.SVC.__init__(self)
-
-    clf = ChildSVC()
-    clf.fit(iris.data, iris.target)
-    clf.predict(iris.data[-1])
-    clf.decision_function(iris.data[-1])
```

Comment:
Apparently the old SVM class hierarchy didn't admit inheritance because attributes were being deleted; see beda3f47. Since the hierarchy was completely refactored since then, I've no strong objections to removing the test, but I don't mind keeping it either.